cmd/server/assets/codes/bulk-issue.html

```diff
@@ -90,6 +92,22 @@
             </small>
           </div>
 
+          {{if $currentRealm.SMSTextAlternateTemplates}}
+          <div class="form-group">
+            <div class="input-group">
+              <select class="form-control" id="sms-template">
+                <option value="Default SMS template">Default SMS template</option>
```

Comment:
there's a subtle thing here where you can't label another one as "Default SMS template"

Do we enforce that restriction in code that this is an invalid label?